index.html

```diff
@@ -0,0 +1,390 @@
+                        </div>
+                    </div>
+                    <div class="latest-works__content">
+                        <img class="latest-works__content__img" src="img/latest-works__2.png">
+                        <div class="latest-works__content__shadow"></div>
+                        <div class="latest-works__content__text">
+                            <p class="latest-works__content__title">Lorem ipsum dolor sit</p>
+                            <p class="latest-works__content__subtitle">ui/ux design</p>
+                        </div>
+                    </div>
+                    <div class="latest-works__content">
+                        <img class="latest-works__content__img" src="img/latest-works__3.png">
+                        <div class="latest-works__content__shadow"></div>
+                        <div class="latest-works__content__text">
+                            <p class="latest-works__content__title">Lorem ipsum dolor sit</p>
+                            <p class="latest-works__content__subtitle">ui/ux design</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="latest-works__info">
+                    <div class="latest-works__content">
+                        <img class="latest-works__content__img" src="img/latest-works__3.png">
+                        <div class="latest-works__content__shadow"></div>
+                        <div class="latest-works__content__text">
+                            <p class="latest-works__content__title">Lorem ipsum dolor sit</p>
+                            <p class="latest-works__content__subtitle">ui/ux design</p>
+                        </div>
+                    </div>
+                    <div class="latest-works__content">
+                        <img class="latest-works__content__img" src="img/latest-works__1.png">
+                        <div class="latest-works__content__shadow"></div>
+                        <div class="latest-works__content__text">
+                            <p class="latest-works__content__title">Lorem ipsum dolor sit</p>
+                            <p class="latest-works__content__subtitle">ui/ux design</p>
+                        </div>
+                    </div>
+                    <div class="latest-works__content">
+                        <img class="latest-works__content__img" src="img/latest-works__2.png">
+                        <div class="latest-works__content__shadow"></div>
+                        <div class="latest-works__content__text">
+                            <p class="latest-works__content__title">Lorem ipsum dolor sit</p>
+                            <p class="latest-works__content__subtitle">ui/ux design</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="pricing-plan">
```

Comment:
Necessary to fix problem with dispersed blocks on the big size screen

Reply:
done